cryptoprotocol/src/main/java/de/upb/crypto/incentive/cryptoprotocol/model/User.java

```diff
@@ -0,0 +1,64 @@
+package de.upb.crypto.incentive.cryptoprotocol.model;
+
+import de.upb.crypto.incentive.cryptoprotocol.model.keys.provider.ProviderPublicKey;
+import de.upb.crypto.incentive.cryptoprotocol.model.keys.user.UserKeyPair;
+import de.upb.crypto.math.serialization.StringRepresentation;
+import de.upb.crypto.math.serialization.annotations.ReprUtil;
+import de.upb.crypto.math.structures.rings.zn.Zn;
+import de.upb.crypto.math.structures.rings.zn.Zn.ZnElement;
+import lombok.Data;
+
+@Data
+public class User
+{
+    private UserKeyPair userKeyPair;
+    private Token userToken;
+
+    /**
+     * (effectively a) no args constructor, generating new key pair upon instantiation. Used when new session in app is started.
```

Comment:
no args but with args :-)

Reply:
Suggestion delete the first part of the sentence.

Reply:
Removed user